packages/contracts-versions/README.md

```diff
@@ -16,15 +16,35 @@ yarn add @aragon/osx-versions
 
 ## Usage
 
+Import Active contracts:
+
+```javascript
+// import active contracts from a specific version
+import {v1_2_0_active_contracts} from '@aragon/osx-versions';
+
+const mumbaiActiveContracts = v1_2_0_active_contracts.mumbai;
+```
+
+Import a specific contract source code from a specific version:
+
+```javascript
+
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity 0.8.17;
+
+// import legacy contracts from a specific version
+import {DAO} from '@aragon/osx-versions/versions/v1_0_1/contracts/core/dao/DAO.sol';
+
+.....
+```
+
+### Generate typechain
```

Comment:
How do we import the bytecode and/or the TS artifacts to test things from Node?

Reply:
i am not sure i understand!
but when u import some source code from contracts-versions/ and you build your contracts for example using hardhat, 
bytecode && artifacts will be build for u inside `/artifacts/@aragon/osx-versions/..../.` ...etc

Reply:
So, my point is to have an example similar to the solidity example import right above. Having an example TS import of a random factory, so that we easily see where the import path is